chart/cas-cif/templates/cronJobs/db-init.yaml

```diff
@@ -66,10 +66,75 @@ spec:
                 - -c
                 - |
                   set -euo pipefail;
-{{ .Values.db.preInitCommand | indent 18 }}
                   create-user-db -d $(CIF_DATABASE) -u $(CIF_USER) -p $(CIF_PASSWORD) --owner;
                   alter-role $(CIF_USER) createrole;
                   psql -d $(CIF_DATABASE) -v "ON_ERROR_STOP=1"<<EOF
                     create extension if not exists postgres_fdw;
                     grant usage on foreign data wrapper postgres_fdw to $(CIF_USER);
                   EOF
+---
+{{- if .Values.db.preInitCommand }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "cas-cif.fullname" . }}-db-pre-upgrade
```

Comment:
should we call it `{{ fullname }}-db-pre-init` and have this template in its own file?